lua/lvim/plugins.lua

```diff
@@ -272,3 +276,14 @@ return {
     disable = not lvim.builtin.terminal.active,
   },
 }
+
+if os.getenv "LVIM_DEV" ~= "1" then
```

Comment:
We should probably create a global variable instead, just so we have more flexibility with how to enable dev-mode.

Reply:
I'm hesitant to introduce more globals, but I do agree it's more flexible that way. Although I'd like there to be "one source of truth" for this, not several.

Should we add a global for test environment as well?

Reply:
> Should we add a global for test environment as well?

We shouldn't need a test variable as long as there are no issues with impatient (regardless if it's enabled or not), so you can replace that test completely with new tests for the new updater mode.

Reply:
I ended up using it to not display the progress indicator (since it spams lines only in the GitHub logs, but not for users due to using `\r`), so I re-added it as a global in a similar manner